vendor-extra/JatsContentBundle/src/EventListener/BuildView/FigCaptionTitleFigureListener.php

```diff
@@ -6,14 +6,14 @@
 
 use FluentDOM\DOM\Element;
 use Libero\ViewsBundle\Views\ConvertsChildren;
-use Libero\ViewsBundle\Views\OptionalTemplateListener;
 use Libero\ViewsBundle\Views\TemplateView;
+use Libero\ViewsBundle\Views\ViewBuildingListener;
 use Libero\ViewsBundle\Views\ViewConverter;
 
 final class FigCaptionTitleFigureListener
 {
     use ConvertsChildren;
-    use OptionalTemplateListener;
+    use ViewBuildingListener;
```

Comment:
also this one (I think) not a case of separation of template and view, as I can see it produces a `TemplateView` with `@LiberoPatterns/figure.html.twig` at the template

Reply:
Input and output of this Listener:
```
<jats:title>Dinaledi skeletal specimens.</jats:title>'
```
```
Libero\ViewsBundle\Views\TemplateView::__set_state(array(
   'arguments' =>  
  array (
    'level' => 3,
    'text' =>  
    array (
      0 =>  
      Libero\ViewsBundle\Views\StringView::__set_state(array(
         'string' => 'Dinaledi skeletal specimens.',
         'context' =>  
        array (
          'area' => 'primary',
          'level' => 3,
          'lang' => 'en',
          'dir' => 'ltr',
        ),  
      )), 
    ),  
  ),  
   'template' => '@LiberoPatterns/heading.html.twig',
   'context' =>                                                                                                                                                            
  array (
    'area' => 'primary',
    'level' => 3,
    'lang' => 'en',
    'dir' => 'ltr',
  ),  
))
```